lib/rules/block-indentation.js

```diff
@@ -81,6 +83,18 @@ function isControlChar(char) {
   return char === '~' || char === '{' || char === '}';
 }
 
+function get(node, path) {
```

Comment:
Can you add a jsdoc comment to this helper function explaining what exactly it's for, maybe providing a basic example of the usage.